libs/components/indicators/src/lib/modules/label/label.component.ts

```diff
@@ -21,10 +21,10 @@ export class SkyLabelComponent {
   public set labelType(value: SkyLabelType | undefined) {
     this.#_labelType = value;
 
-    if (this.#_labelType === undefined) {
+    if (this.labelType === undefined) {
       this.labelTypeOrDefault = LABEL_TYPE_DEFAULT;
     } else {
-      this.labelTypeOrDefault = this.#_labelType;
+      this.labelTypeOrDefault = this.labelType;
```

Comment:
This is needed for 100% code coverage of the `SkyLabelComponent`.

I'm not sure how this wasn't causing issues before now.

Reply:
Could you back this one out? I am already addressing it in a separate PR.